### PR TITLE
OCPQE-14727: skip health-check

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh [hunk 1]
```diff
@@ -208,6 +208,14 @@ function check_history() {
 }
 
 ###Main###
+if [ -f "${SHARED_DIR}/cluster-type" ] ; then
+    CLUSTER_TYPE=$(cat "${SHARED_DIR}/cluster-type")
+    if [[ "$CLUSTER_TYPE" == "osd" ]] || [[ "$CLUSTER_TYPE" == "rosa" ]] || [[ "$CLUSTER_TYPE" == "hypershift-guest" ]]; then
```
Comment (yasun1) [inline ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh:213]: We do not have the 'cluster_type="hypershift-guest"'.  Till now, the hypershift cluster is a kind of rosa cluster. The cluster_type is the product_id of the cluster.
`PRODUCT_ID=$(rosa describe cluster -c "${CLUSTER_ID}" -o json | jq -r '.product.id')`

Comment (yasun1) [reply ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh:213]: Is it overwritten in your steps?

Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh:213]: https://github.com/openshift/release/blob/294c68c2ce708519cd19b8dc352f79c003d79c4d/ci-operator/step-registry/cucushift/hypershift-extended/enable-guest/cucushift-hypershift-extended-enable-guest-commands.sh#L21
@yasun1

Comment (yasun1) [reply ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh:213]: ok

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh:213]: Actually in prow, we are not encouraged to use `if else`, we are supposed to call the needed step in the chain or workflow. So I suggest to remove the health check chain in these ci jobs.

Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh:213]: @jianlinliu You mean we should have a chain: `openshift-e2e-test-hypershift-qe` right? 
cc @liangxia

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh:213]: Yeah, that also works to create a new e2e chain in which does not have the common cluster health check step.

**Comment by LiangquanLi930** [comment]
close this PR, Replace with https://github.com/openshift/release/pull/38172

**Pull request closed without merging**